src/pages/Login/Login.js

```diff
@@ -1,7 +1,134 @@
 import React from 'react';
+import styled from 'styled-components';
+import { ImBubble } from 'react-icons/im';
+import { SiNaver, SiFacebook } from 'react-icons/si';
+import { MdEmail } from 'react-icons/md';
 
 const Login = () => {
-  return <div></div>;
+  return (
+    <LoginContainer>
+      <LoginWrapper>
+        <Emoji>👋</Emoji>
+        <Header>반갑습니다!</Header>
+        <Article>여행의 모든 것, 아워트립</Article>
+        <KakaoBtn>
+          <KakaoBubble />
+          <KakaoParagraph>카카오로 바로 시작</KakaoParagraph>
+        </KakaoBtn>
+        <SocialLogin>
+          <Facebook>
+            <SiFacebook />
+            <CursorParagraph>페이스북</CursorParagraph>
+          </Facebook>
+          <Naver>
+            <SiNaver />
+            <CursorParagraph>네이버</CursorParagraph>
+          </Naver>
+          <Email>
+            <MdEmail />
+            <CursorParagraph>이메일</CursorParagraph>
+          </Email>
+        </SocialLogin>
+        <SignupMessage>
+          <Paragraph>아직 회원이 아니신가요?</Paragraph>
+          <CursorBorderParagraph>회원가입</CursorBorderParagraph>
+        </SignupMessage>
+      </LoginWrapper>
+    </LoginContainer>
+  );
 };
+const LoginContainer = styled.div`
+  ${props => props.theme.flex.flexBox};
+  height: 100vh;
+`;
+
+const LoginWrapper = styled.div`
+  ${props => props.theme.flex.flexBox('column')};
+  width: 430px;
+  height: 450px;
+  border: 1px solid #e9ecef;
+`;
+
+const Emoji = styled.div`
+  margin-bottom: 2.5rem;
+  font-size: 3rem;
+  animation: rotate_image 0.4s linear 4 alternate;
+  transform-origin: 50% 50%;
+
+  @keyframes rotate_image {
+    100% {
+      transform: rotate(90deg);
+    }
+  }
+`;
+
+const Header = styled.header`
+  margin-bottom: 0.7rem;
+  font-size: 1.8rem;
+  font-weight: ${props => props.theme.fontWeight.bold};
+`;
+
+const Article = styled.article`
+  margin-bottom: 2.5rem;
+  color: ${props => props.theme.colors.gray};
+  font-weight: ${props => props.theme.fontWeight.bold};
+`;
+
+const KakaoBtn = styled.button`
+  ${props => props.theme.flex.flexBox('', 'center', 'space-around')};
+  width: 20rem;
+  height: 3rem;
+  padding: 0 80px;
+  margin-bottom: 2.5rem;
+  border: none;
+  border-radius: 0.2rem;
+  background-color: ${props => props.theme.colors.kakaoYellow};
+  font-size: 1rem;
+  font-weight: ${props => props.theme.fontWeight.bold};
+`;
+
+const KakaoBubble = styled(ImBubble)`
+  margin-bottom: 4px;
+`;
+
+const KakaoParagraph = styled.p``;
+
+const SocialLogin = styled.div`
+  display: flex;
+  width: 17rem;
+  justify-content: space-between;
+  margin-bottom: 2.5rem;
+  color: ${props => props.theme.colors.gray};
```

Comment:
구조 분해 할당 할 수 있습니다.
아래 부분도 같이 적용 시켜 보세요!
```suggestion
  color: ${({theme}) => theme.colors.gray};
```

Reply:
props를 전부 ({theme})으로 바꿨습니다!